### DISCUSSION TITLE
Does secondary app support messaging and crashlytics?

### DISCUSSION THREAD

**Discussion by sergeymorkovkin** [invertase/react-native-firebase#4825] [Q&A] [answered]
It's not clear, since there is a lot of contradicting information.

1. No information in official docs - https://rnfirebase.io/app/usage#secondary-apps

2. https://github.com/invertase/react-native-firebase/issues/3337
> Mar 25, 2020 - Those modules don't support secondary apps sorry, that's a Firebase SDK limitation.
> Mar 26, 2020 - That's not possible with the native SDKs I'm afraid.

So, not possible?

3. https://github.com/invertase/react-native-firebase/pull/1864#issuecomment-580236827
> Jan 30, 2020 - I'm going to close this PR for now as stale, this functionality exists on v6 also.

So, it exists?

**Comment by sergeymorkovkin** [comment, marked as answer]
Answered here:
https://stackoverflow.com/questions/62253691/firebasemessaging-getinstancefirebaseapp-for-secondary-app-supposed-to-be-publ

**Reply by mikehardy** [reply]
Officially not supported per that excellent link:

> Answer regarding separate projects for FCM and Crashlytics:

>>    Upon discussing with the team, they’ve verified that your use case is not supported by Firebase. You should be using a single Firebase Project/Firebase App for both FCM and Crashlytics. Thanks for understanding.
